hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -1010,15 +1029,16 @@ public void setOwner(final Path path, final String owner, final String group) th
 
       final AbfsRestOperation op = client.setOwner(getRelativePath(path),
               transformedOwner,
-              transformedGroup);
+              transformedGroup, tracingContext);
```

Comment:
The format needs to be followed. If parameters are being mentioned one per line, same needs to be followed. Applies to other places in this PR too.

Reply:
done